resources-generator/src/main/kotlin/dev/icerock/gradle/generator/ColorsGenerator.kt

```diff
@@ -72,9 +72,9 @@ abstract class ColorsGenerator(
     protected open fun getPropertyModifiers(): Array<KModifier> = emptyArray()
     protected open fun getPropertyInitializer(color: ColorNode): CodeBlock? {
         val className = if (color.isThemed()) {
-            "Themed(light = Color(0x${color.lightColor}), dark = Color(0x${color.darkColor}))"
+            "Themed(light = Color(0x${color.lightColor}), dark = Color(0x${color.darkColor}), name = \"${color.name}\")"
         } else {
-            "Single(color = Color(0x${color.singleColor}))"
+            "Single(color = Color(0x${color.singleColor}), name = \"${color.name}\")"
```

Comment:
here we should improve logic to support apple & android native way of colors usage. i describe it in `ColorResource` file comment

Reply:
fixed in #439